src/main/java/com/example/tour/rating/RatingController.java

```diff
@@ -0,0 +1,31 @@
+package com.example.tour.rating;
+
+import com.example.tour.TransformerUtils;
+import com.example.tour.activity.Activity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/rating")
+
+public class RatingController {
+    private final RatingService ratingService;
+
+    @Autowired
+    public RatingController(RatingService ratingService) {
+        this.ratingService = ratingService;
+    }
+    @GetMapping
+    public List<RatingDTO> getAllRating(){
+        return TransformerUtils.createListRatingDTO(ratingService.getRatings());
+    }
+
+    @PostMapping
+    public void addNewRating(@RequestBody Activity activity, Rating rating) {
+        if (activity == null || activity.isActive()) return;
+        ratingService.addNewRating(rating);
+    }
```

Comment:
(now I remembered you said this endpoint was not working yet)

This code takes the fields of an Activity from a POST request, a Rating from nobody knows where, and adds a rating to the db only passing the rating? I don't get it. Does it work? (ignore this last question)